charts/taraxa-node/templates/transaction-generation-service.yaml

```diff
@@ -0,0 +1,73 @@
+{{ if .Values.transactionGeneration.enabled }}
```

Comment:
lets rename file to: `transaction.generation.statefulset.yaml`. I have another PR, which cleans and renames files.